patches/api/0180-Mob-Spawner-API-Enhancements.patch

```diff
@@ -4,16 +4,190 @@ Date: Fri, 19 Apr 2019 12:41:19 -0500
 Subject: [PATCH] Mob Spawner API Enhancements
 
 
-diff --git a/src/main/java/org/bukkit/block/CreatureSpawner.java b/src/main/java/org/bukkit/block/CreatureSpawner.java
-index cb447a4ad5a9dce7c98999a5d7fcd6111fc9b10e..5bbae759ce39d42886994e500fd9454ec328f804 100644
---- a/src/main/java/org/bukkit/block/CreatureSpawner.java
-+++ b/src/main/java/org/bukkit/block/CreatureSpawner.java
-@@ -199,4 +199,30 @@ public interface CreatureSpawner extends TileState {
-      * @see #getSpawnRange()
-      */
-     public void setSpawnRange(int spawnRange);
+diff --git a/src/main/java/io/papermc/paper/SpawnerHolder.java b/src/main/java/io/papermc/paper/SpawnerHolder.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..f8c6975509655b59c739aa0e634e0f3c2e6a72b2
+--- /dev/null
++++ b/src/main/java/io/papermc/paper/SpawnerHolder.java
+@@ -0,0 +1,202 @@
++package io.papermc.paper;
++
++import org.bukkit.entity.EntityType;
++import org.jetbrains.annotations.NotNull;
++
```

Comment:
misses a class comment like "represents something that can handle spawner information" or something
(I bet you can come up with something better than me, lol)